### DISCUSSION TITLE
Is there by any chance any support for elecrow ESP32 7" Display ?

### DISCUSSION THREAD

**Discussion by mikrocoder** [Bodmer/TFT_eSPI#2932] [New processor/display support or capability request]
Hi,

would your lib work with this display as well? [elecrow esp32-display-7](https://www.elecrow.com/esp32-display-7-inch-hmi-display-rgb-tft-lcd-touch-screen-support-lvgl.html )
This seems to have an exotic Driver IC EK9716BD3 and EK73002ACGB?  Or are they compatible with another?

**Comment by Bodmer** [comment]
No, that display interface is not supported.

I would stick to LVGL for that display.
